front/src/pages/EditItem/EditItems.tsx

```diff
@@ -0,0 +1,158 @@
+// file was renamed from "EditItem.tsx" to "EditItems.tsx"
+// cause: typescript didn't work if we had the previous name
+
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import styles from './EditItem.module.scss';
+import useFetchTodos from '../../hooks/useFetchTodos';
+import { importanceValues } from '../../constants';
+import { todosAPI } from '../../api/todos-api';
+
+const EditItem = () => {
+  const { todoId } = useParams();
+  const {
+    data: todos = []
+  } = useFetchTodos();
+  const currentTodo = todos.find(todo => todo.id === todoId);
+  const [todoState, setTodoState] = useState({
+    title: currentTodo?.title || '',
+    importance: currentTodo?.importance || '',
+    isDone: currentTodo?.isDone || false,
+  })
+  const navigate = useNavigate();
+
+  let pageTitle: string;
+  let pageContent: JSX.Element;
+  if (currentTodo) {
```

Comment:
let's split this into 2 different components

there is no profit to keep them merged into single one, and for maintainability there is only harm

Reply:
@skyboyer 
I just removed todo list. There is no necessity for it on EditItem page.